synapse/tests/files/stormpkg/testpkg.yaml

```diff
@@ -22,6 +22,24 @@ docs:
     - title: Foo Bar
       path: docs/foobar.md
 
+perms:
+    - perm: [ power-ups, testpkg, user ]
+      gate: cortex
+      desc: Controls user access to testpkg.
+      workflowconfig: true
+
+configvars:
+    - name: API key
+      varname: testpkg:apikey
+      desc: API key to use for querying the testpkg API.
+      workflowconfig: true
```

Comment:
What does workflowconfig currently denote?

Reply:
Whether there should be a workflow for configuring the var. Could just skip it and say that anything in these lists gets a workflow.